README.md

```diff
@@ -123,3 +123,25 @@ To organize and steer the development of [helm-operator-plugins](https://github.
 
 - Slack: [##operator-sdk-dev](https://kubernetes.slack.com/messages/operator-sdk-dev)
 - [Mailing list](https://groups.google.com/forum/#!forum/operator-framework) (Subscribe to access the agenda document.)
+
+## Ansible Working Group
+
+To organize and steer the development of Ansible-based Operators and related projects:
+* [Ansible-based Operators](https://github.com/operator-framework/operator-sdk)
+* [operator_sdk.util Ansible collection](https://github.com/operator-framework/operator-sdk-ansible-util)
+* [kubernetes.core Ansible collection](https://github.com/ansible-collections/kubernetes.core)
+
+### Meetings
+
+* Community Meeting: [Third Monday at 12:00 ET (Eastern Time)](https://docs.google.com/document/d/1Bo68RhJkpr0Nk_JpkBdbkgmi3c2WtfdvFZ0jVSJzvXo/edit?usp=sharing) (monthly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=12:00&tz=ET%20%28Eastern%20Time%29).
```

Comment:
```suggestion
* Community Meeting: [Third Monday at 12:00 PM ET (Eastern Time)](https://docs.google.com/document/d/1Bo68RhJkpr0Nk_JpkBdbkgmi3c2WtfdvFZ0jVSJzvXo/edit?usp=sharing) (monthly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=12:00&tz=ET%20%28Eastern%20Time%29).
```

Reply:
Ignore, looks like we're using 24 hour format. https://github.com/operator-framework/community/blob/master/README.md#meetings-1 so 12:00 ET would indicate noon.